src/sql/workbench/contrib/notebook/browser/notebook.contribution.ts

```diff
@@ -344,6 +344,19 @@ configurationRegistry.registerConfiguration({
 	}
 });
 
+configurationRegistry.registerConfiguration({
+	'id': 'notebookViews',
+	'title': 'Notebook Views',
```

Comment:
Should it be localized even though it is a proper name? I also didn't localize it because Notebook and other similar settings weren't either, but if it still makes sense to you, I can do that.

Reply:
"Notebook" could be left alone - but Views I'm assuming isn't part of the proper name here. If it is though then it's fine to leave that, but please indicate that in a comment.

Reply:
I asked Julie about this, but she didn't have an answer right now. Localized this for now.